app/src/main/java/com/teixeira/vcspace/screens/editor/EditorScreen.kt

```diff
@@ -188,7 +188,7 @@ private fun configureColorScheme(editor: VCSpaceEditor) {
           "Abyss" -> "abyss"
           "Solarized Dark" -> "solarized_drak"
           "Python Dark Mode" -> "pythondm"
-          else -> if ((followSystemTheme && isDarkTheme) || isDarkMode) "darcula" else "quietlight"
+          else -> if ((followSystemTheme && isDarkTheme) || isDarkMode) "pythondm" else "quietlight"
```

Comment:
Okay. It was just a personal prefrence because it is my own repo. Amd for the remember last opened file, can you just set it to true and user can't change it? I mean who doesn't want his code to open back